runtime/syntax/editorconfig.vim

```diff
@@ -1,15 +1,19 @@
+" Vim syntax file
+" Language:     EditorConfig
+" Maintainer:   Gregory Anders <greg@gpanders.com>
```

Comment:
Do we want to keep the "maintainer" paradigm for syntax/runtime files which are fully owned by Neovim (instead of ported from Vim)?

Personally I think I'd rather this file be "owned" by the project/organization, particularly considering that EditorConfig is a builtin, first class feature of Neovim (opposed to other syntax files, which are a "best effort" thing).

cc @clason @seandewar (<-- runtime/funtime team)

```suggestion
```

Reply:
I agree; I just requested _some_ sort of header. I'd be fine with this simply being 
```vim
Nvim syntax file
```

Reply:
Personally I'd be fine with that.